### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/cwp-agencyextensions/actions?query=branch%3Apulls%2F2.6%2Fmodule-standards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3"
     },
+    "conflict": {
+        "silverstripe/auditor": "<2.4.0"
```
Comment (ChloeSartorelli) [inline composer.json]: Why is this conflict added? If it's the same issue that we've dealt with in PRs that were merged already, please update to the correct version.

File: composer.json [hunk 2]
```diff
@@ -17,6 +17,9 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3"
```
Comment (ChloeSartorelli) [inline composer.json:18]: Travis config requires fulltextsearch - should we add that here?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Updated

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json:18]: There's no skipped unit tests and there's no mention of fulltextsearch in the codebase, so I think we can safely omit it

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**